chapters/appendix-VIII-SPDX-Lite.md

```diff
@@ -0,0 +1,52 @@
+# Appendix-VIII SPDX Lite
+
+## 1. Explanation of SPDX Lite 
+
+The SPDX Lite defines the subset of the SPDX specification, from the point of view of use cases in industries. The SPDX Lite aims at the balance between the SPDX standard and actual workflows in industries. 
+
+The SPDX Lite is the subset of the SPDX specification. The SPDX Lite consists of mandatory part of the Document Creation and Package Information and other basic information. 
+
+The mandatory part of the Package information in the SDPX Lite is basic but indispensable for complying with licenses. It is easy to understand licensing information by reading an SPDX Lite file. It is easy to create manually an SPDX Lite file by anyone who does not have enough knowledge about licensing infomation, so that tools are not necessarily required to create an SPDX Lite file. 
+
+The SPDX Lite has the affinity with SPDX tools due to its containing the mandatory part of the Document Creation and Package Information in the SDPX Lite. 
+
+An SPDX Lite file can be used parallel with an SPDX file in software supply chains. 
+
+
+## 2. format of SPDX Lite
+
+The SPDX Lite is the subset of the SPDX specification. The SPDX Lite consists of mandatory part of the Document Creation and Package Information and other basic information. 
+
+The mandatory part of the Document Creation (those are Document Name, SPDX Document Namespace and Creator) is used for keeping the compatibility with SPDX tools.
+
+The main part of the Package Information (those are Package Name,  Package Version, Package File Name, Package Download Location,  Package Home Page, Concluded License, Declared License, Comments on License and Copyright Text) is used for exchanging license information.
+
+In the Package Information, Package SPDX Identifier and Files Analyzed are used for keeping the compatiblity with SPDX tools. 
+
+File Analyzed must be set to "false", when SPDX Lite is used.
+
+Other information (License Identifier, Extracted Text, License Name and License Comment) is used for exchanging license information.
+
+
+## Table of SPDX Lite
+
+| # | corresponding SPDX section no. | License Info. | Tag  | Rationale      |
```

Comment:
I am not sure whether the Rationale column is necessary. Is the idea to explain the rationale for including them in SPDX Lite? If so, then several of them might need more details.

Or if the idea is just to explain generally what the field is for, I don't think that's necessary because the main SPDX spec already does that.